docs/_layouts/global.html

```diff
@@ -17,113 +19,133 @@
+                        </div>
+                    </li>
+
+                    <li class="nav-item dropdown">
+                        <a href="#" class="nav-link dropdown-toggle" id="navbarDeploying" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Deploying</a>
+                        <div class="dropdown-menu" aria-labelledby="navbarDeploying">
+                            <a class="dropdown-item" href="cluster-overview.html">Overview</a>
+                            <a class="dropdown-item" href="submitting-applications.html">Submitting Applications</a>
+                            <div class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="spark-standalone.html">Spark Standalone</a>
+                            <a class="dropdown-item" href="running-on-mesos.html">Mesos</a>
+                            <a class="dropdown-item" href="running-on-yarn.html">YARN</a>
+                            <a class="dropdown-item" href="running-on-kubernetes.html">Kubernetes</a>
+                        </div>
+                    </li>
+
+                    <li class="nav-item dropdown">
+                        <a href="#" class="nav-link dropdown-toggle" id="navbarMore" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More</a>
+                        <div class="dropdown-menu" aria-labelledby="navbarMore">
+                            <a class="dropdown-item" href="configuration.html">Configuration</a>
+                            <a class="dropdown-item" href="monitoring.html">Monitoring</a>
+                            <a class="dropdown-item" href="tuning.html">Tuning Guide</a>
+                            <a class="dropdown-item" href="job-scheduling.html">Job Scheduling</a>
+                            <a class="dropdown-item" href="security.html">Security</a>
+                            <a class="dropdown-item" href="hardware-provisioning.html">Hardware Provisioning</a>
+                            <a class="dropdown-item" href="migration-guide.html">Migration Guide</a>
+                            <div class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="building-spark.html">Building Spark</a>
+                            <a class="dropdown-item" href="https://spark.apache.org/contributing.html">Contributing to Spark</a>
+                            <a class="dropdown-item" href="https://spark.apache.org/third-party-projects.html">Third Party Projects</a>
+                        </div>
+                    </li>
+
+                    <li class="nav-item">
+                        <input type="text" id="docsearch-input" placeholder="Search the docs…">
+                    </li>
+                </ul>
+                <!--<span class="navbar-text navbar-right"><span class="version-text">v{{site.SPARK_VERSION_SHORT}}</span></span>-->
+            </div>
+        </nav>
+
+        {% if page.url == "/" %}
```

Comment:
To give the intro page a bit more spice, I'm using the hero banner as on the main webpage with the colors. This only appears when the index page is called.